charts/agent-operator/templates/operator-deployment.yaml

```diff
@@ -40,6 +40,10 @@ spec:
         resources:
         {{- toYaml . | nindent 10 }}
         {{- end }}
+        args:
```

Comment:
Does this still work if there are no arguments?

Reply:
Will check

Reply:
Didn't need to check!